lua/typescript-tools/tsserver_provider.lua

```diff
@@ -158,7 +154,7 @@ function TsserverProvider:get_executable_path()
 
   if mason_tsserver and not tsserver_exists(tsserver_path) then
     local _ = log.trace() and log.trace("tsserver", tsserver_path:absolute(), "not exists.")
-    tsserver_path = mason_tsserver:joinpath("typescript", "lib", "tsserver.js")
+    tsserver_path = mason_tsserver:join("typescript", "lib", "tsserver.js")
```

Comment:
I think it should be:
```suggestion
    tsserver_path = mason_tsserver:joinpath("typescript", "lib", "tsserver.js")
```